src/pages/workspaces/workspace/analysis/modals/ComputeModal.test.js

```diff
@@ -21,28 +21,6 @@ import {
 } from 'src/pages/workspaces/workspace/analysis/runtime-utils'
 
 
-jest.mock('src/components/Modal', () => {
-  const { div, h } = jest.requireActual('react-hyperscript-helpers')
-  const originalModule = jest.requireActual('src/components/Modal')
-  return {
-    ...originalModule,
-    __esModule: true,
-    default: props => div({ id: 'modal-root' }, [
-      h(originalModule.default, { onAfterOpen: jest.fn(), ...props })
-    ])
-  }
-})
-
-// Mocking PopupTrigger to avoid test environment issues with React Portal's requirement to use
-// DOM measure services which are not available in jest environment
-jest.mock('src/components/PopupTrigger', () => {
```

Comment:
WorkspaceMenu.test.js also contains this `PopupTrigger` mocking. Is it no longer needed?

Reply:
It looks like WorkspaceMenu.test.js mocks MenuTrigger from PopupTrigger to render the menu always open instead of requiring a click to open. The tests fail without it.